src/components/SortMovie/SortMovie.tsx

```diff
@@ -1,34 +1,41 @@
 import style from './Sortmovie.module.scss'
-import { useDispatch } from 'react-redux'
-import { changeFilterParam, changeSortParam } from '../../store/actions/sort'
-
+import { useSearchParams } from 'react-router-dom'
 
 const SortMovie = () => {
+  const [searchParams, setSearchParams] = useSearchParams();
 
-  const dispatch = useDispatch()
+  const elementCarriage = document.querySelector('.sort_carriage')
```

Comment:
you should avoid directly trying to access to dom elements, instead consider `refs`